src/video_core/renderer_vulkan/vk_rasterizer.cpp

```diff
@@ -469,25 +469,12 @@ bool Rasterizer::BindResources(const Pipeline* pipeline) {
             }
         }
 
-        for (const auto& desc : info.texture_buffers) {
-            if (!desc.is_written) {
-                const VAddr address = desc.GetSharp(info).base_address;
-                meta_read = texture_cache.IsMeta(address);
-            }
-        }
-
         if (!meta_read) {
             for (const auto& desc : info.buffers) {
                 if (IsMetaUpdate(desc)) {
                     return false;
                 }
             }
-
-            for (const auto& desc : info.texture_buffers) {
-                if (IsMetaUpdate(desc)) {
-                    return false;
```

Comment:
The lambda IsMetaUpdate is not needed anymore and can be inlined into the loop for simplicity sake